src/content/concepts/configuration.md

```diff
@@ -45,8 +46,12 @@ module.exports = {
 };
 ```
 
+_See_: [Configuration section](/configuration) for the all supported config options
+
 ## Multiple Targets
 
+Along with exporting a single configuration object/function, webpack supports exporting multiple configurations.
```

Comment:
__webpack supports exporting multiple configurations__ - this line makes me think that webpack will export the configurations, when that is user's responsibility to write a configuration module with export statement.

suggest rewording into: webpack can accept multiple configurations

Reply:
Fair enough 👍🏽